prae/src/lib.rs

```diff
@@ -79,6 +79,7 @@ pub use prae_macro::define;
 
 #[cfg(test)]
 mod test_deps {
+    use assert_matches as _;
     use serde as _;
     use serde_json as _;
 }
```

Comment:
Btw, why is this necessary?

Reply:
I mean the whole `test_deps` business

Reply:
I added the lint to warn about unused crate dependencies.

```rust
#![warn(unused_crate_dependencies)]
```

But it give's false positives when buliding the separate integration test binaries.

```bash
❯ cargo test
   Compiling prae_macro v0.2.1 (/home/brianh/Projects/third_party/prae/prae_macro)
   Compiling prae v0.4.5 (/home/brianh/Projects/third_party/prae/prae)
warning: external crate `assert_matches` unused in `prae`: remove the dependency or add `use assert_matches as _;`
   |
note: the lint level is defined here
  --> prae/src/lib.rs:71:9
   |
71 | #![warn(unused_crate_dependencies)]
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^
   = help: remove unnecessary dependency `assert_matches`

warning: external crate `serde` unused in `prae`: remove the dependency or add `use serde as _;`
  |
  = help: remove unnecessary dependency `serde`

warning: external crate `serde_json` unused in `prae`: remove the dependency or add `use serde_json as _;`
  |
  = help: remove unnecessary dependency `serde_json`

warning: 3 warnings emitted

    Finished test [unoptimized + debuginfo] target(s) in 0.73s
     Running unittests (target/debug/deps/prae-dc495198ddde61f3)
```

Reply:
Oh, now I get it. I'll add a comment referencing the related issue: https://github.com/rust-lang/rust/issues/57274

Reply:
Nice :wink: